pkg/gui/gui.go

```diff
@@ -869,6 +871,72 @@ func (gui *Gui) showIntroPopupMessage() {
 	})
 }
 
+func (gui *Gui) showBreakingChangesMessage() {
+	_, err := types.ParseVersionNumber(gui.Config.GetVersion())
+	if err != nil {
+		// We don't have a parseable version, so we'll assume it's a developer
+		// build, or a build from HEAD with a version such as 0.40.0-g1234567;
+		// in these cases we don't show release notes.
+		return
+	}
```

Comment:
@jesseduffield I'm actually wondering if we need this check. We could omit it, in that case people coming from an older release and switching to a HEAD build for the first time would see the release notes that have accumulated so far, once. They will never see release notes again until they switch back to an official release, so I'm not sure it's necessary to support this case, but less code is always better.

What do you think?

Reply:
I prefer us only showing the release notes when you're on an actual numbered version. Reason being that we're likely to append to a single version's breaking changes message several times as we merge PR's that count towards that one release. If we show users the message prematurely, they won't be able to see the final product. I'm pretty sure that's true.

Reply:
I see, that scenario makes sense, so I'm keeping this condition.

> That user would eventually switch back to a numbered version.

Actually that's kind of unlikely once they saw how much better the master version is. :-)